nrrd/reader.py

```diff
@@ -423,9 +425,8 @@ def read(filename, custom_field_map=None):
     """
 
     """Read a NRRD file and return a tuple (data, header)."""
-
     with open(filename, 'rb') as fh:
         header = read_header(fh, custom_field_map)
         data = read_data(header, fh, filename)
 
-        return data, header
+        return data, header 
```

Comment:
Would remove extra space at the end of this line.

Any whitespace at the end of lines are not necessary